apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/BigQueryClientTest.cs

```diff
@@ -74,7 +79,7 @@ protected override object GetArgument(ParameterInfo parameter)
                 return new TableReference();
             }
             return base.GetArgument(parameter);
-        }
+        }        
```

Comment:
I wish I knew what I did (keyboard wise) that kept doing this. I'm sure I didn't used to.